calendar_event_jitsi/models/calendat_event.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
+
+from odoo import models, fields, api
+from odoo.addons.http_routing.models.ir_http import slug
+
+
+class Meeting(models.Model):
+    _inherit = 'calendar.event'
+
+    jitsi_meet_url = fields.Char(
+        string="Jitsi Meet URL",
+        compute='_compute_jitsi_meet_url',
+        store=True,
+        help=(
+            "Jitsi Meet URL"
+        ),
+    )
+
+    @api.depends("name")
```

Comment:
Not sure I like the fact that it's a computed method that depends on "name" to recompute.

Take this scenario, for example:

1. User creates meeting, and sends invitation to Customer
2. User changes name of the meeting
..
3. User will try to connect to the new meeting url, but Customer will connect to the old meeting url.


I think we should generate the meeting url on `create`, and never change it again. Do you agree?
Or, easiest solution, simply `api.depend('id')`, that will never change

Reply:
@ivantodorovich .. I have remove the computed field and added normal field with readonly=True.

And also override the "create" method to assign jitsi meet url to this field.

NOTE : we can not use api.depend('id') becasue Compute method cannot depend on field 'id'